persistence/commons/src/main/java/org/apache/causeway/persistence/commons/integration/repository/RepositoryServiceDefault.java

```diff
@@ -99,6 +105,27 @@ public <T> T detachedEntity(final @NonNull T entity) {
         return factoryService.detachedEntity(entity);
     }
 
+    @Override
+    public <T> T execInBulk(Callable<T> callable, Class<?>... classes) {
+        try {
+            Arrays.stream(classes).forEach(aClass -> setBulkMode(aClass, Boolean.TRUE));
+            return callable.call();
+        } catch (Exception e) {
```

Comment:
I don't think this should swallow the exception... it should be propagated so that the transaction is aborted if there's a problem.

Reply:
Yes

Reply:
Fixed